tensornetwork/tests/tensornetwork_test.py

```diff
@@ -640,6 +645,10 @@ def test_weakref(backend):
     e.node2
 
 
+# This test no longer tests weakref behaviour
```

Comment:
Weak refs are still in the code though right?

Reply:
Yes, still there